client/my-sites/github-deployments/components/repositories/create-repository/index.tsx

```diff
@@ -30,40 +28,69 @@ export const CreateRepository = () => {
 		page( indexPage( siteSlug! ) );
 	};
 
+	const canUserCreateDeployment = useSelector( ( state ) =>
```

Comment:
I think we can get rid of these checks, then, because of https://github.com/Automattic/wp-calypso/pull/87741#top